go/Connection/TypeDBDriverImpl.go

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package Connection
+
+import "C"
+import (
+	"typedb_driver/go/api/database"
+	"typedb_driver/go/api/user"
+	"typedb_driver/go_wrapper"
+)
+
+type TypeDBDriver struct {
+	nativeObject typedb_driver.Connection
+	databaseMgr database.DatabaseManager
+	userMgr    user.UserManager
+}
+
+func NewTypeDBDriverwithString(address string) *TypeDBDriver {
```

Comment:
Why is it called this way? `New` ... `withString`...?

Reply:
Is it a convention for constructors in `go`? `with` should be capitalized, I guess?

Reply:
I left this in by accident - I previously had 2 constructors so I needed to clarify.